src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -646,7 +645,6 @@ public void testRegisterUser() throws InterruptedException, ApiException {
 
     // Verify that the correct Sam API calls were made
     verify(samUsersApi).createUserV2(null);
-    verify(samTosApi).acceptTermsOfService(TOS_URL);
```

Comment:
Earlier in the test we still have this mock, could you remove it as well?

```
when(samTosApi.acceptTermsOfService(anyString())).thenReturn(userStatus);
```
I think this would have failed if strict stubs were working properly.  If you have the capacity to port this test over to JUnit 5 that would be awesome.

Reply:
Oh that's a good point! Thanks for catching that!

Reply:
I ported the all of the tests in this file over to JUnit 5. It was a bigger change than I anticipated because there are a lot of mocks that only some of the tests use, but I did catch some unnecessary stubbing!